stanford-sw/test/src/edu/stanford/LocationTestsCurriculum.java

```diff
@@ -0,0 +1,83 @@
+package edu.stanford;
+
+import java.io.IOException;
+
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.apache.solr.client.solrj.SolrServerException;
+import org.junit.Before;
+import org.junit.Test;
+import org.marc4j.marc.*;
+import org.xml.sax.SAXException;
+
+/**
+ * junit4 tests for location facet for locked stacks
+ * @author Shelley Doljack
+ */
+public class LocationTestsCurriculum extends AbstractStanfordTest
```

Comment:
it's actually a JUnit practice to call test classes xxxTests.   https://blog.testproject.io/2017/04/16/test-automation-best-practices/